xml/tuning-intro.xml

```diff
@@ -0,0 +1,366 @@
+   </itemizedlist>
+   <para>
+     Software:
+   </para>
+   <itemizedlist>
+     <listitem>
+       <para>
+         &productname; &prevproductnumber;
+       </para>
+     </listitem>
+     <listitem>
+       <para>
+         &sls; 15 SP1
+       </para>
+     </listitem>
+   </itemizedlist>
+   <note>
+     <para>
+       Limited use subscriptions are provided with &productname;
+       as part of the subscription entitlement
+     </para>
+   </note>
+   <sect2>
+     <title>Performance Metrics</title>
+     <para>
+       The performance of storage is measured on two different, but related
+       axis: <literal>latency</literal> and <literal>throughput</literal>. In
+       some cases, one is more important than the other. An example of
+       <literal>throughput</literal> being more
+       important is that of backup use cases, where <literal>throughput</literal> is the most
+       critical measurement and maximum performance is achieved with larger
+       transfer sizes. Conversely, for a high performance database, <literal>latency</literal>
+       is the most important measurement.
+     </para>
+     <para>
+       <literal>Latency</literal> is the time from when the request was made to when it was
+       completed. This is usually measured in terms of milliseconds. This
+       performance can be directly tied to CPU clock speed, the system bus,
+       and device performance.
+       <literal>Throughput</literal> is the measure of an amount of data that can be written
+       or retrieved within a particular time period. The measure is usually
+       in MB/s, GB/s, or perhaps MiB/s, and GiB/s.
+     </para>
+     <para>
+       A third measurement that is often referred to is <literal>IOPS</literal>.
+       This stands for Input/output Operations Per Second. This
+       measure is somewhat ambigous as the result is reliant on the size of
+       the I/O operation, what type (read/write), and details about the I/O
+       pattern: fresh write, overwrite, random write, mix of reads and writes.
+       While ambigous, it is still a valid tool to use when measuring
+       the changes in the performance of your storage environment. For
+       example, it is possible to make adjustments that may not affect the
+       latency of a single 4k sequential write operation, but would allow
+       many more of the operations to happen in parallel, resulting in a
+       change in throughput and IOPS.
+     </para>
+   </sect2>
+ </sect1>
+ <sect1 xml:id="tuning-measurement">
+   <title>Determining What to Measure</title>
+   <para>
+     When tuning an environment, it is important to understand the I/O that
+     is being tuned for. By properly understanding the I/O pattern, it is
+     then possible to match the tests to the environment, resulting in a
+     close simulation of the environment.
+   </para>
+   <para>
+     Tools that can be useful for understanding the I/O patterns are:
+   </para>
+   <itemizedlist>
```

Comment:
Should the tool names be wrapped in [command] tags?
And should they not have punctuation after them? 
Value judgement here I think -- that is why I am asking, not suggesting.

Reply:
:thinking: I actually hadn't thought about it. I'll re-review. Cheers

Reply:
clarify however is appropriate and improves readability.